### PR TITLE
URW-169 Update activities page

### PR DESCRIPTION
**Pull request by Chenneth**
Information was either
a) poorly worded (e.g. "companies coming out", "these companies" [when the wording of "such as" implies that there are more, but usage of "these" sounds like we are only referring to the 3 example comapnies given]; etc.)
b) not up-to-date (e.g. "Our collegiate team just came back"; botathon being on April 29)

Also adjusted the Botathon section. This way it sells better than "an intro to robotics". And there's now an href to the botathon page (I know it's in the nav bar, but I still say we should do that so they don't have to look for the page at all.)

### CODE REVIEW

**Comment by Chenneth** [comment]
I will add that this page is really bad; to compare it to an essay, there's a lot of evidence, but it doesn't mean anything. I'm not saying we need an entire book or anything, but listing things out like "UNT Robo did this once", and "UNT Robo did that another time" doesn't exactly sound interesting. It comes off as though we did these things solely so we could put it down that we were there. I'd suggest having these not listed like that and somehow merged into a singular paragraph.

Alternatively, do URW-136 and don't fix the issue with listing. It might look better if there's actual substance surrounding the list.

These also need an _actual_ description. What does "where Bell Flights (Formerly Bell Helicopters) came out to talk" mean? What did they talk about? Did they talk about their favorite socks? Or was it actual robotics and aviation? And "we discussed the best way to win a Hackathon" is really informal. I'd suggest rewording it to "we provided suitable strategies for tackling any hackathon" or something similar rather than "here's how you win!" because that's sending the message that we think winning is all hackathons are about, when the main thing is making hack-y things, hanging out with friends, and getting to compete and code. I'm not changing these because I don't know anything about those events, so I may misrepresent the event. 

**Also we need to fix that header image.  The only one there that is still decently associated with Robotics is Lauren and Tyler.** If this image was for a competition team, then by all means leave it, but from my point of view, I have no idea who these people are, and I have no idea what that metal box in the bottom-middle is supposed to be.

**Comment by Chenneth** [comment]
I say "In addition to the STEM companies' seminar" but that still sounds awkward. I'm trying to keep the original author's intent by keeping the transition there, but I'm not sure what we had the companies do, nor a decent general grouping for them besides "STEM companies". I do not like the usage of "these" or "those" as that sounds like it refers to only L3, RoboKind, and StandardUser CyberSecurity.

**Comment by Chenneth** [comment]
Apparently this isn't even on the main site

**Comment by TylerAdamMartinez** [comment]
@Chenneth, what site is this for, then?

**Comment by Chenneth** [comment]
The dev site has the activites page but the main doesn't.
